logfire/_internal/exporters/otlp.py

```diff
@@ -38,7 +41,20 @@ def gen() -> Iterable[bytes]:
                         yield chunk
 
                 request.body = gen()  # type: ignore
-        return super().send(request, **kwargs)
+
+        max_attempts = 7
```

Comment:
Should this be configurable in any way?

Reply:
I don't think so. Making it configurable would be painful with little benefit, and OTEL's retrying of status codes isn't configurable.